src/init.sh

```diff
@@ -68,57 +68,56 @@ function init_kw()
   notification_config_file_template="${KW_ETC_DIR}/notification_template.config"
   remote_file_template="${KW_ETC_DIR}/remote.config"
 
-  if [[ -f "$config_file_template" && -f "$build_config_file_template" ]]; then
-    mkdir -p "$PWD/$KW_DIR"
-    cp "$config_file_template" "$PWD/$KW_DIR/$name"
-    cp "$vm_config_file_template" "${PWD}/${KW_DIR}/${vm_name}"
-    cp "$build_config_file_template" "${PWD}/${KW_DIR}/${build_name}"
-    cp "$deploy_config_file_template" "${PWD}/${KW_DIR}/${deploy_name}"
-    cp "$mail_config_file_template" "${PWD}/${KW_DIR}/${mail_name}"
-    cp "$notification_config_file_template" "${PWD}/${KW_DIR}/${notification_name}"
-    cp "$remote_file_template" "${PWD}/${KW_DIR}/${remote_name}"
-
-    sed -i -e "s/USERKW/$USER/g" -e '/^#?.*/d' "$PWD/$KW_DIR/${vm_name}"
-    sed -i -e "s,SOUNDPATH,$KW_SOUND_DIR,g" -e '/^#?.*/d' "$PWD/$KW_DIR/${notification_name}"
-
-    if [[ -n "${options_values['ARCH']}" ]]; then
-      if [[ -d "$PWD/arch/${options_values['ARCH']}" || -n "${options_values['FORCE']}" ]]; then
-        set_config_value 'arch' "${options_values['ARCH']}" "${PWD}/${KW_DIR}/${build_name}"
-      elif [[ -z "${options_values['FORCE']}" ]]; then
-        complain 'This arch was not found in the arch directory'
-        complain 'You can use --force next time if you want to proceed anyway'
-        say 'Available architectures:'
-        find "$PWD/arch" -maxdepth 1 -mindepth 1 -type d -printf '%f\n' | sort -d
-      fi
-    fi
+  if [[ ! -f "$config_file_template" || ! -f "$build_config_file_template" ]]; then
+    complain "No such: $config_file_template"
+    exit 2 # ENOENT
+  fi
 
-    if [[ -n "${options_values['REMOTE']}" ]]; then
-      populate_remote_info "${options_values['REMOTE']}"
-      if [[ "$?" == 22 ]]; then
-        complain 'Invalid remote:' "${options_values['REMOTE']}"
-        exit 22 # EINVAL
-      else
-        set_config_value 'ssh_user' "${remote_parameters['REMOTE_USER']}"
-        set_config_value 'ssh_ip' "${remote_parameters['REMOTE_IP']}"
-        set_config_value 'ssh_port' "${remote_parameters['REMOTE_PORT']}"
-      fi
+  mkdir -p "$PWD/$KW_DIR"
+  cp "$config_file_template" "$PWD/$KW_DIR/$name"
+  cp "$vm_config_file_template" "${PWD}/${KW_DIR}/${vm_name}"
+  cp "$build_config_file_template" "${PWD}/${KW_DIR}/${build_name}"
+  cp "$deploy_config_file_template" "${PWD}/${KW_DIR}/${deploy_name}"
+  cp "$mail_config_file_template" "${PWD}/${KW_DIR}/${mail_name}"
+  cp "$notification_config_file_template" "${PWD}/${KW_DIR}/${notification_name}"
+  cp "$remote_file_template" "${PWD}/${KW_DIR}/${remote_name}"
```

Comment:
Could you create another commit that addresses these comments:
1. Use string concatenation.
2. Use cmd_manager instead of the command.
3. Enable verbose mode.

Or, at least, create an issue for tracking these requests.

P.s.: This comment is valid for the entire file.